src/smart-components/Conversation/hooks/useHandleChannelEvents.js

```diff
@@ -63,6 +63,26 @@ function useHandleChannelEvents({ currentGroupChannel, sdkInit, hasMoreToBottom
         }
       };
 
+      ChannelHandler.onReadReceiptUpdated = (channel) => {
```

Comment:
Why do we have to `type: messageActions.SET_CURRENT_CHANNEL` here?

Reply:
We need to set the current channel when the receipts change, and SET_CURRENT_CHANNEL exactly do that